resources/keycloak/custom-user-provider/src/main/java/com/cvmanager/auth/provider/user/DbUtil.java

```diff
@@ -0,0 +1,25 @@
+package com.cvmanager.auth.provider.user;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import org.keycloak.component.ComponentModel;
+import static com.cvmanager.auth.provider.user.CustomUserStorageProviderConstants.*;
+
+public class DbUtil {
```

Comment:
Question: This question spans multiple files (`DbUtil`, `CustomUserStorageProvider`, and `CustomUserStorageProviderFactory`). Would it make more sense to move this single database connection method into the `CustomUserStorageProvider `or somewhere else? It seems like the intent of the `CustomUserStorageProvider` (see [close()](https://github.com/CDOT-CV/jpo-cvmanager/pull/85/files#diff-3337e5fbea3fb73084e2c44e9bd44d7b9a7e67eebec15478c21bb640dead1e4eR104)) is to encapsulate the database interactions with the User table. I see that we use the connection to validate the configuration in `CustomUserStorageProviderFactory`, but I'm not sure that warrants moving the connection management into a generic "util" class

Reply:
Moved into CustomUserStorageProvider